src/molara/Structure/structure.py

```diff
@@ -102,15 +103,16 @@ def calculate_bonds(self: Structure | Crystal | Molecule) -> np.ndarray:
         vdw_radii = np.array([atom.vdw_radius for atom in self.atoms])
         coordinates = np.array([atom.position for atom in self.atoms])
 
-        for i in range(len(self.atoms)):
-            atom1_coord = coordinates[i]
-            atom1_radius = vdw_radii[i]
+        max_distance = 2.0 * vdw_radii.max() / 1.75
```

Comment:
Could we make these variables that can be changed in a user setting window?

Reply:
I could set a class attribute (`self.bond_distance_factor=1./1.75`)

I think that implementing the Gui dialog for customising the display settings is an issue of its own...

Reply:
Agreed.